pkg/services/alerting/notifiers/pagerduty.go

```diff
@@ -88,8 +100,17 @@ func (pn *PagerdutyNotifier) buildEventPayload(evalContext *alerting.EvalContext
 		eventType = "resolve"
 	}
 	customData := simplejson.New()
-	for _, evt := range evalContext.EvalMatches {
-		customData.Set(evt.Metric, evt.Value)
+	if !pn.MessageInDetails {
```

Comment:
Could you please flip these if statements? it's preferable to avoid using negation.